sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -181,18 +181,60 @@ Rendered input:
 
 ![Apify Actor input schema - country input](./images/input-schema-country.png)
 
+Example of date selection using absolute and relative `datepicker` editor:
+
+```json
+{
+    "absolute_date": {
+        "title": "Date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker"
+    },
+    "relative_date": {
+        "title": "Relative date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker",
+        "allowAbsolute": false,
+        "allowRelative": true
+    },
+    "any_date": {
+        "title": "Any date",
+        "type": "string",
+        "description": "Select date",
+        "editor": "datepicker",
+        "allowRelative": true
+    }
+}
+```
+
+Rendered input for `absolute_date` property. See that by default, the date picker allows selecting only an absolute date in the form `YYYY-MM-DD`:
+
+![Apify Actor input schema - country input](./images/input-schema-date-absolute.png)
+
+Rendered input for `relative_date` property in form of `+/- {number} {unit}`. (The `allowAbsolute` is explicitly set to `false`):
```

Comment:
```suggestion
The `relative_date` property renders an input field that accepts dates in the `+/- {number} {unit}`. nthe `allowAbsolute` parameter is set to `false` to restrict input to relative dates only.
```

Reply:
IMHO, to make this clear, we should write that 

> ...renders an input field that enables the user to choose the relative date and returns the value in the format `+/- {number} {unit}`, for example, `+ 2 days`.

This is a documentation for Actor developers so we need to focus on what is returned by the field which is what they work with.